assets/css/form-style.css

```diff
@@ -50,6 +50,8 @@ form input {
     opacity: 0;
     transition: opacity 0.25s ease;
     border-radius: 8px;
+    left: 50%;
+    transform: translateX(-50%);
```

Comment:
This is an excellent solution to the problem, and it's much better than what I tried to do before. There's a couple of issues still present that appear to be due to my previous attempts at centering these container divs. You can see this in the login page and its error notifications. These overlays still appear to be off-center, and that's because of `left: calc((100vw - 30vw) / 2);` on line 69 for the notification overlay, and `setScalableWidth` and `setScalableHeight` functions inside `util-frontend.js` (they set the `left` and `top` properties respectively).

I'll try to help out when I get the chance, but I believe that if you take a look at these sections, and make sure that only your change above is setting the `left` position of the overlay, then all overlays should appear centered.

I think you can also add `top: 50%` and change `transform` to be `translate(-50%, -50%)` and it will center the container both horizontally and vertically.

Let me know if you have any questions.

Reply:
Sure thing, will work on updating this as well!